src/index.mako.html

```diff
@@ -163,6 +164,18 @@
     <div ng-controller="GaSeoController">
       <div ga-seo ga-seo-options="options" ga-seo-map="map"></div>
     </div>
+% if device == 'embed':
+    <div style="display: none;">
+      <div class="topic-container" ng-controller="GaTopicController">
+        <div ga-topic ga-topic-ui="select" ga-topic-options="options" ga-topic-map="map"></div>
+      </div>
+      <div ng-controller="GaTranslationController">
+        <div ga-translation-selector ga-translation-selector-options="options"></div>
+      </div>
+      <div ga-background-layer-selector ga-background-layer-selector-map="map"></div>
+    </div>
+% endif
```

Comment:
@gjn here it's the directives which are necessary for that the app works correctly